src/services/stringCompletions.ts

```diff
@@ -674,36 +723,61 @@ namespace ts.Completions.StringCompletions {
             }
         }
 
-        return result;
+        return arrayFrom(result.values());
+    }
+
+    function getPatternFromFirstMatchingCondition(target: unknown, conditions: readonly string[]): [string] | undefined {

```

Comment:
Doesn't this need changes to support fetching pattern results from nested conditions?

Reply:
Indeed, I forgot about this. Updated, thanks 👍